CODE_STYLE.md

```diff
@@ -63,9 +63,30 @@
 ## Fortran Example
 
 ```Fortran
+
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL Flexible Modeling System (FMS).
+!*
+!* FMS is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FMS is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+!***********************************************************************
+
 !> @file
 !! @brief Example code
 !! @author <developer>
```

Comment:
Does anyone know what is meant by <developer> here? Many files in FMS have their emails inserted in this space.

Reply:
We can slowly remove the personal email address of the GFDL developers, and add the gfdl.climate.model.info@noaa.gov email as the one single email.  If external developers want to include their email, they are more than welcome to do so.